test/test_format.py

```diff
@@ -0,0 +1,528 @@
+# from .helper import session
+from mathics.session import MathicsSession
+
+session = MathicsSession()
+
+# from mathics.builtin.base import BoxConstruct, Predefined
+
+
+import pytest
+
+
+# MathML StandardForm
+@pytest.mark.parametrize(
+    ("str_expr", "str_expected", "msg"),
+    [
+        ('"4"', "<mtext>4</mtext>", None),
+        # The following seems wrong: they should be formatted as numbers
+        ("4", "<mn>4</mn>", None),
+        ('"4.32"', "<mtext>4.32</mtext>", None),
+        ('"4.32313213213`2"', "<mtext>4.32313213213`2</mtext>", None),
```

Comment:
This behavior is wrong: the output should be ``<mtext>4.32`2</mtext>``.

Reply:
If the `('"4.32313213213`2"', "<mtext>4.32313213213`2</mtext>", None),` is wrong we shouldn't be testing for wrong output. 

We can do something like

```
TODO_TESTS=[('"4.32313213213`2"', "<mtext>4.32`2</mtext>", None), ...] 
```

and remove the wrong test from the list.